metric/bigtable/src/main/java/com/spotify/heroic/metric/bigtable/BigtableConnectionBuilder.java

```diff
@@ -113,22 +136,23 @@ public BigtableConnection call() throws Exception {
             builder.enableEmulator(emulatorEndpoint);
         }
 
-        final BigtableOptions bigtableOptions = builder.build();
+        final var bigtableOptions = builder.build();
```

Comment:
why change all these to vars?

Reply:
personal preference and OCD mainly ¯\_(ツ)_/¯ 

But in all seriousness, if you get a newcomer to look at the before & after code, I think the majority will prefer the vars because the code is so much cleaner. All those LHS types are just noise.

Reply:
also, the IDE makes the LHS types clickable so that they “pop” to a var just by clicking on them. It’s kind of addictive…